bigquery_etl/stripe/charge.schema.json

```diff
@@ -1136,10 +1136,6 @@
     "type": "RECORD",
     "description": "undocumented card",
     "fields": [
-      {
-        "name": "id",
-        "type": "STRING"
-      },
```

Comment:
this schema needed to be sorted so that it matched `data.event` in `stripe_external.events_v1`